enterprise/checklist.md

```diff
@@ -7,23 +7,25 @@ keywords: lists todo checklist
 controls: toolbar button
 ---
 
-# Overview
-
 A checklist is a proven method to improve productivity and quality of a business and are generally a great way of getting things done. The **Checklist** plugin helps the user keep track of all required actions by creating a simple, clean, and easy to use TODO list.
 
 Common use cases for using a checklist:
 
-*   **Private & Work** - Checklist can help in keeping work and private tasks separate for more efficiency.
-*   **Tracking Bills** - Checklists can be created to avoid forgetting payment cycles to pay a bill on time.
-*   **Shopping list** - Checklist can help create a grocery list in the browser that can be shared with the other family members to collaborate or act as a reminder while shopping.
-*   **Entertainment tracker** - Checklist can help create list of movies & TV series to keep a tab on the movies to watch or set weekly reminders at the right time to avoid missing a show.
-*   **Industry applications** - There are a lot of professional applications of using a Checklist, such as:
-    *   Used in quality assurance of software engineering to check process compliance, code standardization, and error prevention.
-    *   Pre-flight checklists to aid in aviation safety to ensure that critical items are not overlooked.
-    *   Used by some investors as a critical part of their investment process.
-    *   Checklists are useful for school camps to create emergency survival kits.
-    *   In professional diving checklists can be used for preparation of equipment for a diving session.
+* **Private & Work** - Checklist can help in keeping work and private tasks separate for more efficiency.
+* **Tracking Bills** - Checklists can be created to avoid forgetting payment cycles to pay a bill on time.
+* **Shopping list** - Checklist can help create a grocery list in the browser that can be shared with the other family members to collaborate or act as a reminder while shopping.
+* **Entertainment tracker** - Checklist can help create list of movies & TV series to keep a tab on the movies to watch or set weekly reminders at the right time to avoid missing a show.
```

Comment:
Should this be 'create **a** list of movies'?

Reply:
lol probably. I only fixed the markup in this section